frontend/src/components/Profile.js

```diff
@@ -49,62 +52,63 @@ class Profile extends React.Component {
 
   didUpdate(event, parent, ref) {
     if(event !== null) {
-      var attendees = ["user1", "user2", "user3"];
-    var len = 0;
-    var imageUrl = "";
-    var card = "";
-
-    // Since images are option field, evaluate whether the field isn't null
-    if (event.imagePaths != null) {
-      len = event.imagePaths.length;
-    }
-    // Retrieve image urls only if they were provided. Otherwise set url to default value
-    if(len > 0) {
-      imageUrl = event.imagePaths.slice(1, len - 2);
-      imageUrl = imageUrl.split(",");
+      var attendees = event.attendees.slice(1, event.attendees.length -1).split(",");
+      var namesOfAttendees = this.getNamesOfAttendees (attendees);
+      var len = 0;
+      var imageUrl = "";
+      var card = "";
 
-      // dynamically create a card that contain images for the event
-      card = imageUrl.map(url => <Card.Img variant="top" src={url} />);
-    }
+      // Since images are option field, evaluate whether the field isn't null
+      if (event.imagePaths != null) {
+        len = event.imagePaths.length;
+      }
+      // Retrieve image urls only if they were provided. Otherwise set url to default value
+      if(len > 0) {
```

Comment:
this is already in an if block for `len > 0`, but should you be checking if `len > 1`?